signac/contrib/project.py

```diff
@@ -789,8 +787,8 @@ def _contains_job_id(self, job_id):
             True if the job id is initialized for this project.
 
         """
-        # We can rely on the project workspace to be well-formed, so just use
-        # str.join with os.sep instead of os.path.join for speed.
+        # We can rely on the project workspace to be well-formed, so use
+        # os.sep.join instead of os.path.join for performance.
         return os.path.exists(os.sep.join((self.workspace(), job_id)))
```

Comment:
```suggestion
        return os.path.exists(os.sep.join((self.workspace, job_id)))
```

Reply:
This probably needs a more global find-and-replace, I'll take care of that.